models/tmp/stg_zendesk__time_zone_tmp.sql

```diff
@@ -1,5 +1,5 @@
 --To disable this model, set the using_schedules variable within your dbt_project.yml file to False.
 {{ config(enabled=var('using_schedules', True)) }}
 
-select * 
-from {{ var('time_zone') }}
+select {{ dbt_utils.star(source('zendesk', 'time_zone')) }} 
+from {{ source('zendesk', 'time_zone') }} as timezone
```

Comment:
If I recall this alias was needed to avoid the BigQuery data structure issue. If that is the case, should we just alias these always to be safe?